float32 board
float32 rail
float32 power_voltage
float32 power_current
float32 leak
float32 adc3

